cypress/integration/Train/ScenarioAndTags.spec.js

```diff
@@ -0,0 +1,149 @@
+import * as models from '../../support/Models'
+import * as modelPage from '../../support/components/ModelPage'
+import * as editDialogModal from '../../support/components/EditDialogModal'
+import * as train from '../../support/Train'
+import * as helpers from '../../support/Helpers'
+
+describe('Training Scenario and Tags', () => {
```

Comment:
Similar to the question in teams. Why do we need this test?  It looks like it's doing very similar things to the existing test and doesn't add missing cases or increase coverage

Reply:
What question in teams? Please use teams to point it out to me. Thanks.

Reply:
What existing test? I am not aware of an existing test for this scenario. This is the test you gave me in skeleton form with the beginning stages completed.

Reply:
https://github.com/Microsoft/ConversationLearner-UI/blob/master/cypress/integration/scenarioAndTags.spec.js

Also, it's not specific to Train Dialogs. It has some assertions about not showing tags or description on Log Dialogs too which is why it doesn't fit in that category.

Reply:
So that file was renamed/deleted so that it fit the naming pattern of all other test spec files.

Actually Log Dialog test stages was something I wanted to ask you, but we've been too busy debating other aspects of test code that I just did not get around to it...

Why are we looking in Log Dialogs for something that should not be there? Is there the potential in the UI product code that could put those fields in the Log Dialog view if there were some bug introduce there?

Reply:
Yes, there is potential for a bug to be introduced. The modal/window used for viewing dialogs is shared between train dialogs and log dialogs so we need to be sure those fields don't show up in certain circumstances such as when viewing a log dialog.

> So that file was renamed/deleted

Yes... and you deleted it.  I have another PR that was going to adjust it so I guess I'll put it back. At the time when I created the link the file existed on `master`, but you merged to master affecting the link. I should have linked to a fixed version:

https://github.com/Microsoft/ConversationLearner-UI/blob/1973bcf0a30a3d39a15cc45ffff534317c9edd38/cypress/integration/scenarioAndTags.spec.js